packages/edit-site/src/components/editor/index.js

```diff
@@ -265,6 +264,15 @@ export default function Editor( { isLoading, onClick } ) {
 
 	return (
 		<>
+			<GlobalStylesRenderer />
+			<EditorKeyboardShortcutsRegister />
+			{ isEditMode && <BlockKeyboardShortcuts /> }
+			{ showVisualEditor && (
+				<>
+					<TemplatePartConverter />
+					<PatternModal />
+				</>
```

Comment:
I think both of these template part converter and pattern modal components need to move to the editor package. They provide features that are site editor only for now for no obvious reasons.

Reply:
`PatternModal` can definitely move IMO and also move the commands that use them. The template part converter though is probably because we do not allow inserting template parts in posts. Now that both editors are converging, I'm curious if the decision to exclude template parts in post editor should be revisited too..

Reply:
We do allow template parts in the template editor which is also available in the post editor. So it should move too.

For the pattern modal, I'm moving it here #61862
For the template part converter, I'm waiting for @jorge's PR that moves the template part creation modal first.

Reply:
Hi @youknowriad, that PR is available at https://github.com/WordPress/gutenberg/pull/61879.